responsys-api.gemspec

```diff
@@ -17,15 +17,15 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
-  spec.required_ruby_version = '>= 1.9.3'
+  spec.required_ruby_version = '>= 2.1.0'
```

Comment:
The CI error is because tests are not based on a Gemfile.lock, but on the gemspec and so pulling the latest nokogiri version and they dropped support for Ruby 1.9 in their latest versions. I'm not sure of the possibility of enforcing a specific nokogiri version just for the 1.9.3 suite, so I'll probably remove it and keep tests running in the Ruby 2 environment. 

But we still don't want to disallow apps to use the GEM if they're not using Ruby 2. I'm not Tim Cook and I don't sell Ruby 1.9 => Ruby 2 adapters.
Apps will be responsible for choosing and locking the nokogiri version they want to use based on the Ruby version they use in their environment, or the nokogiri version that bundles the latest security updates they want to use too.